lib/typescript/components/inputs/Input/style.module.scss

```diff
@@ -0,0 +1,182 @@
+@import 'assets/scss/colors.scss';
+@import 'assets/scss/fonts.scss';
+
+.label {
+  width: 100%;
+}
+
+.input {
+  display: flex;
+  background-color: white;
+  width: 100%;
+  border: 1px solid var(--color-dark-elements-gray);
+  border-radius: 4px;
+  color: var(--color-text-contrast);
+  overflow: hidden;
+
+  &::placeholder {
+    color: var(--color-text-gray);
+  }
+
+  &:focus-within {
+    background-color: var(--color-background-blue);
+    border: 1px solid var(--color-airy-blue);
+    outline: none;
+  }
+}
+
+.inputInvalid {
+  .inputTitle {
```

Comment:
nitpick: Since there is only one input title I think this nesting in `inputValid` is unnecessary.